### PR TITLE
ignore keys from /private/var/folders, delete-keychain after use

### PR DESCRIPTION
**Pull request by jakubgs**
Resolves issue with temporary keychains showing up after `security list-keychains` when signing desktop app:
```
### Adding code-signing keychain to search list...
    "/Users/jenkins/Library/Keychains/login.keychain-db"
    "/private/var/folders/lg/klkwp62n51v5y38020d8x4xc0000h8/T/tmp.yxhfQkBR/macos-developer-id.keychain-db"
    "/private/var/folders/lg/klkwp62n51v5y38020d8x4xc0000h8/T/tmp.D8l0Y0dZ/macos-developer-id.keychain-db"
    "/private/var/folders/lg/klkwp62n51v5y38020d8x4xc0000h8/T/tmp.KgxajoRV/macos-developer-id.keychain-db"
    "/private/var/folders/lg/klkwp62n51v5y38020d8x4xc0000h8/T/tmp.u3uZ5NqH/macos-developer-id.keychain-db"
    "/private/var/folders/lg/klkwp62n51v5y38020d8x4xc0000h8/T/tmp.42KaB6hF/macos-developer-id.keychain-db"
    "/private/var/folders/lg/klkwp62n51v5y38020d8x4xc0000h8/T/tmp.elctVcZE/macos-developer-id.keychain-db"
    "/private/var/folders/lg/klkwp62n51v5y38020d8x4xc0000h8/T/tmp.ii5IHVYZ/macos-developer-id.keychain-db"
    "/Library/Keychains/System.keychain"
```
Resolves: https://github.com/status-im/status-react/issues/6352

### CODE REVIEW

**Review by vkjr — APPROVED** [review]

File: scripts/sign-macos-pkg.sh [hunk 1]
```diff
@@ -65,8 +67,8 @@ function clean_up {
   security list-keychains -s $ORIG_KEYCHAIN_LIST
   security list-keychains
 
-  echo -e "\n### Wiping keychain file..."
-  rm -P "$KEYCHAIN"
+  echo -e "\n### Delete keychain file..."
+  security delete-keychain "$KEYCHAIN"
```
Comment (mandrigin) [inline scripts/sign-macos-pkg.sh:71]: this should also happen on errors, I think that is what caused initial issues

**Review by mandrigin — CHANGES_REQUESTED** [review]
one small thing, looks good otherwise

**Review by vitvly — APPROVED** [review]

File: scripts/sign-macos-pkg.sh (hunk 1 above)
Comment (jakubgs) [reply scripts/sign-macos-pkg.sh:71]: It does happen on errors. See:
https://github.com/status-im/status-react/pull/7272/files#diff-7020266fb517002c883a982991d1b155R80
```
trap clean_up EXIT
```

**Review by mandrigin — APPROVED** [review]

File: scripts/sign-macos-pkg.sh (hunk 1 above)
Comment (jakubgs) [reply scripts/sign-macos-pkg.sh:71]: Although, maybe we should make it to be:
```
trap clean_up EXIT ERR
```
Just in case.

**Comment by mandrigin** [comment]
thanks!

**Review by pedropombeiro — APPROVED** [review]

**Pull request merged**